docs/modules/ROOT/examples/cloudscale_objectsuser.yaml

```diff
@@ -1,7 +1,13 @@
-apiVersion: cloudscale.s3.appcat.vshn.io/v1
+apiVersion: cloudscale.crossplane.io/v1
 kind: ObjectsUser
 metadata:
   name: my-cloudscale-user
-  namespace: default
 spec:
-  secretRef: my-cloudscale-user-credentials
+  forProvider:
+    secretRef:
+      name: my-cloudscale-user-credentials
+      namespace: default
+    tags:
+      key: value
+  providerConfigRef:
```

Comment:
Wouldn't it be better to get the `providerConfig` from the Provider CRD instead of having this info multiplied in all ObjectsUser?

Reply:
Seems like this is a design choice from Crossplane.